src/qt/createwalletdialog.cpp

```diff
@@ -32,7 +32,7 @@ CreateWalletDialog::CreateWalletDialog(QWidget* parent) :
         // set to true, enable it when isEncryptWalletChecked is false.
         ui->disable_privkeys_checkbox->setEnabled(!checked);
 #ifdef ENABLE_EXTERNAL_SIGNER
-        ui->external_signer_checkbox->setEnabled(!checked);
+        ui->external_signer_checkbox->setEnabled(m_has_signers && !checked);
```

Comment:
Maybe
```suggestion
        if (m_has_signers) {
            ui->external_signer_checkbox->setEnabled(!checked);
        }
```
to make the same code structure as it is in `CreateWalletDialog::setSigners` for better readability?